empmgmt-servlet/src/main/java/com/neha/empmgmt/dao/impl/EmployeeDaoImpl.java

```diff
@@ -14,58 +16,172 @@
 
 	@Override
 	public Employee findById(int id) {
+		Employee employee = new Employee();
 		String sql = "Select * from employee where id = ?";
 		PreparedStatement stmt = null;

```

Comment:
Ok.I did the formatting changes and commited. Are you able to see the changes?If no, then I will create the new pull request.